### ISSUE TITLE
Bundle size: Moment package is large and barely used 

### ISSUE THREAD

**Issue by 1cgonza** [forensic-architecture/timemap#213]
We are seeking to reduce the bundle size and Moment seems to be one that could potentially be replaced with Vanilla JS. For our version we did this to remove the package since it is only used in `src/common/utilities.js`:

```js
export function calcDatetime(date, time) {
  if (!time) time = '00:00';
  const formats = {
    date: DATE_FMT.split('/'),
    time: TIME_FMT.split(':'),
  };
  const d = date.split('/');
  const t = time.split(':');
  const dI = formats.date.indexOf('DD');
  const mI = formats.date.indexOf('MM');
  const yI = formats.date.indexOf('YYYY');
  const hoursI = formats.time.indexOf('HH');
  const minutesI = formats.time.indexOf('mm');

  // Months go from 0-11, ergo the -1 in months.
  return new Date(d[yI], d[mI] - 1, d[dI], t[hoursI], t[minutesI]);
}
```

This only accounts for very limited time formats at the moment and it is not as smart as Moment when constructing dates which is a pain in JS. @ebefarooqui do you think this is worth pursuing as a PR for future versions or is Moment something being used in many other places now? Our solution just does the work for our case but thought it is a good proof of concept for potentially eliminating Moment as a dependency.

**Comment by breezykermo** [comment]
Picking up in lieu of a response from @ebefarooqui here @1cgonza, apologies for the great delay. We'd be happy to accept a PR that removes moment from the repo in place of slimmer date functions, yes.

**Comment by 1cgonza** [comment]
Hey @breezykermo you are back! sure I can send PR for this. Is `development` de prefered branch to send it to? `main` seems more active now.

**Comment by breezykermo** [comment]
PRs can go direct to main, correct.

**Comment by breezykermo** [comment]
Closed with #234
